docs/proposals/docker-registry-2-4.md

```diff
@@ -0,0 +1,57 @@
+# Support for Docker Registry 2.4
+## Abstract
+
+A proposal to update the docker/distribution to v2.4.0.
+
+## Motivation
+
+Primary goal is to migrate to Schema 2 [1].
```

Comment:
I hope that it is no longer needed too. Now I'm trying to understand how we need the [BlobDeleter](https://github.com/openshift/origin/blob/master/pkg/dockerregistry/server/admin.go) after rebase.